stratisd_cert.py

```diff
@@ -235,6 +236,18 @@ def tearDown(self):
         D-Bus trace.
         :return: None
         """
+        if StratisdCertify.verify_devices is True:
+            try:
+                non_stratis_by_id_devices = []
```

Comment:
Please rename this to something a bit more generic, like "disallowed_syslinks" or something.

Reply:
OK; renamed.  I'm going to let CI run, and then if it passes, squash everything into a single commit.